stories/0-Input.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {Fragment} from "react";
+import {storiesOf} from "@storybook/react";
+
+import FormField from "../src/form/field/FormField";
+import Input from "../src/form/input/Input";
+import StoryFragment from "./utils/StoryFragment";
+
+storiesOf("Input", module)
+  .add("Input", () => (
+    <Fragment>
+      <Input
+        name={"fullName"}
+        onChange={(e) => console.log(e.currentTarget.value)}
+        placeholder={"Input"}
+      />
+
+      <br />
+
+      <Input
+        name={"fullName"}
+        onChange={(e) => console.log(e.currentTarget.value)}
+        placeholder={"Disabled Input"}
+        isDisabled={true}
+      />
+    </Fragment>
+  ))
+  .add("FormField with Input", () => (
```

Comment:
Not a big deal but isn't better `Input with FormField` instead?

Reply:
I agree, it will emphasize the component name better.